stores/src/main/scala/zio/raft/stores/segmentedlog/OpenSegment.scala

```diff
@@ -62,7 +63,7 @@ class OpenSegment[A <: Command: Codec](
   // TODO: we should only write entries that will fit in the segment and return the rest to be written in the next segment
   def writeEntries(entries: List[LogEntry[A]]): ZIO[Any, Nothing, Int] =
     for {
-      bytes <- ZIO.attempt(entriesCodec.encode(entries).require.toByteVector).orDie
```

Comment:
I think this is a mistake... this entriesCodec is not related to RpcMessageCodec